src/server.c

```diff
@@ -6056,6 +6054,8 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "mem_replication_backlog:%zu\r\n", mh->repl_backlog,
             "mem_total_replication_buffers:%zu\r\n", server.repl_buffer_mem + server.repl_full_sync_buffer.mem_used,
             "mem_replica_full_sync_buffer:%zu\r\n", server.repl_full_sync_buffer.mem_used,
+            "mem_asm_migrate_output_buffer:%zu\r\n", mh->asm_migrate_output_buffer,
+            "mem_asm_import_input_buffer:%zu\r\n", mh->asm_import_input_buffer,
```

Comment:
@ShooterIT let me know what you think about this rename. I think it will be better to start with `mem_` and indicating input/output might be more descriptive to people who are not familiar with asm feature.

Reply:
looks good to me